Source.NetCore/project.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+	"supports": {
+		"net46.app": { },
+		"dnxcore50.app": { }
+	},
+	"dependencies": {
+		"Microsoft.NETCore": "5.0.0",
+		"Microsoft.NETCore.Portable.Compatibility": "1.0.0",
+		"NuSpec.ReferenceGenerator": "1.3.6",
+		"System.Diagnostics.StackTrace": "4.0.0",
+		"System.Reflection.Emit": "4.0.0",
+		"System.Reflection.Extensions": "4.0.0",
+		"System.Reflection.TypeExtensions": "4.0.0"
+	},
+	"frameworks": {
+		"dotnet": {
```

Comment:
What about `netstandard`? I know that it not supported yet but de facto it could be used e.g `dotnet5.5`, `dotnet5.4`, etc.